utils/PHPStanAttributeTypeSyncer/src/ValueObject/Paths.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Utils\PHPStanAttributeTypeSyncer\ValueObject;
+
+final class Paths
+{
+    /**
+     * @var string
+     */
+    public const NAMESPACE_PHPDOC_NODE = 'Rector\AttributeAwarePhpDoc\Ast\PhpDoc';
+
+    /**
+     * @var string
+     */
+    public const NAMESPACE_NODE_FACTORY = 'Rector\AttributeAwarePhpDoc\AttributeAwareNodeFactory';
```

Comment:
I wish there was a PHP `::namespace` syntax (like `::class`) when I see this. ;)
Here's two alternatives : 

```php
<?php
$namespace = (new \ReflectionClass(SomeClassInThatNamespace::class))->getNamespaceName();
$namespace = (string) preg_replace('/(\\\(?!.*\\\).*)/', '', SomeClassInThatNamespace::class);
```

Reply:
Cool :+1: 
, I'll try to wrap it in some function.

Reply:
Ha, I tried it, but in this case it would assume there is some class in the namespace. While there is 0 classes before generating and they could be overriden/removed later.

I'll stick with namespaces for now.